### ISSUE TITLE
Does not support multiple enclosures

### ISSUE THREAD

**Issue by TingPing** [vslavik/winsparkle#51]
If you have an item such as;

``` xml
<item>
    <enclosure sparkle:os="windows-x86" />
    <enclosure sparkle:os="windows-x64" />
</item>
```

It will always overwrite the values with the last enclosure. It could store a list of these enclosures or maybe only store the first enclosure that is for the correct platform.

**Comment by vslavik** [comment]
Related: #50.

Yes, the best matching enclosure should be used, and the check for `<item>`’s suitability should consider all enclosures.

**Comment by bjorn** [comment]
:+1:

**Comment by abnerlee** [comment]
So how to provide x86 and x64 update correctly? Should I write appcast like this?

```
<item>
    <enclosure sparkle:os="windows-x86" />
</item>
<item>
    <enclosure sparkle:os="windows-x64" />
</item>
```

**Comment by vslavik** [comment]
@abnerlee 
Step 1: Implement fix
Step 2: Submit PR
Step 3: Enjoy the warm fuzzy feeling of helping the world *and* fixing your issue.

**Comment by vslavik** [comment]
Fixed by #283.
